terraform/examples/io500/run_io500-sc22.sh

```diff
@@ -139,109 +117,112 @@ format_storage() {
   export LD_LIBRARY_PATH=/usr/local/mpifileutils/install/lib64/:$LD_LIBRARY_PATH
 
-  log "Prepare config file 'temp.ini' for IO500"
+  log.info "Prepare config file 'temp.ini' for IO500"
 
   # Set the following vars in order to do envsubst with config-full-sc21.ini
   export DAOS_POOL="${DAOS_POOL_LABEL}"
   export DAOS_CONT="${DAOS_CONT_LABEL}"
   export MFU_POSIX_TS=1
-  export IO500_NP=$(( ${DAOS_CLIENT_INSTANCE_COUNT} * $(nproc --all) ))
+  export IO500_NP=$(( DAOS_CLIENT_INSTANCE_COUNT * $(nproc --all) ))
 
-  # cp -f "${IO500_DIR}/config-full-sc21.ini" .
-  # envsubst < config-full-sc21.ini > temp.ini
+  # shellcheck disable=SC2153
   envsubst < "${IO500_INI}" > temp.ini
+  sed -i "s|^datadir.*|datadir = ${IO500_DATAFILES_DFUSE_DIR}|g" temp.ini
   sed -i "s|^resultdir.*|resultdir = ${IO500_RESULTS_DFUSE_DIR}|g" temp.ini
   sed -i "s/^stonewall-time.*/stonewall-time = ${IO500_STONEWALL_TIME}/g" temp.ini
   sed -i "s/^transferSize.*/transferSize = 4m/g" temp.ini
-  #sed -i "s/^blockSize.*/blockSize = 1000000m/g" temp.ini # This causes failures
   sed -i "s/^filePerProc.*/filePerProc = TRUE /g" temp.ini
   sed -i "s/^nproc.*/nproc = ${IO500_NP}/g" temp.ini
 
   # Prepare final results directory for the current run
   TIMESTAMP=$(date "+%Y-%m-%d_%H%M%S")
   IO500_RESULTS_DIR_TIMESTAMPED="${IO500_RESULTS_DIR}/${TIMESTAMP}"
-  log "Creating directory for results ${IO500_RESULTS_DIR_TIMESTAMPED}"
+  log.info "Creating directory for results ${IO500_RESULTS_DIR_TIMESTAMPED}"
   mkdir -p "${IO500_RESULTS_DIR_TIMESTAMPED}"
 }
 
 run_io500() {
-  log_section "Run IO500"
   mpirun -np ${IO500_NP} \
     --hostfile "${SCRIPT_DIR}/hosts_clients" \
     --bind-to socket "${IO500_DIR}/io500" temp.ini
-  log "IO500 run complete!"
+  log.info "IO500 run complete!"
 }
 
 show_pool_state() {
-  log "Query pool state"
+  log.info "Query pool state"
   dmg pool query "${DAOS_POOL_LABEL}"
 }
 
 process_results() {
-  log_section "Copy results from ${IO500_RESULTS_DFUSE_DIR} to ${IO500_RESULTS_DIR_TIMESTAMPED}"
+  log.info "Copy results from ${IO500_RESULTS_DFUSE_DIR} to ${IO500_RESULTS_DIR_TIMESTAMPED}"
 
   cp config.sh "${IO500_RESULTS_DIR_TIMESTAMPED}/"
   cp hosts* "${IO500_RESULTS_DIR_TIMESTAMPED}/"
 
   echo "${TIMESTAMP}" > "${IO500_RESULTS_DIR_TIMESTAMPED}/io500_run_timestamp.txt"
 
-  FIRST_SERVER=$(echo ${SERVER_LIST} | cut -d, -f1)
-  ssh ${FIRST_SERVER} 'daos_server version' > \
+  FIRST_SERVER=$(echo "${SERVER_LIST}" | cut -d, -f1)
+  ssh "${FIRST_SERVER}" 'daos_server version' > \
     "${IO500_RESULTS_DIR_TIMESTAMPED}/daos_server_version.txt"
 
   RESULT_SERVER_FILES_DIR="${IO500_RESULTS_DIR_TIMESTAMPED}/server_files"
+  # shellcheck disable=SC2013
   for server in $(cat hosts_servers);do
     SERVER_FILES_DIR="${RESULT_SERVER_FILES_DIR}/${server}"
     mkdir -p "${SERVER_FILES_DIR}/etc/daos"
     scp "${server}":/etc/daos/*.yaml "${SERVER_FILES_DIR}/etc/daos/"
+    scp "${server}":/etc/daos/*.yml "${SERVER_FILES_DIR}/etc/daos/"
```

Comment:
Should probably be done in this way, if you want to have the file glob expansion properly done on the ssh target:
```suggestion
    scp "${server}:/etc/daos/*.yml" "${SERVER_FILES_DIR}/etc/daos/"
```